presto-main/src/main/java/io/prestosql/metadata/Metadata.java

```diff
@@ -243,7 +243,7 @@
     /**
      * Begin insert query
      */
-    InsertTableHandle beginInsert(Session session, TableHandle tableHandle);
+    InsertTableHandle beginInsert(Session session, TableHandle tableHandle, List<ColumnHandle> columnHandles);
```

Comment:
cmt msg:
> `Skip adding null during Insert if the column has default value`

This commit title is wrong, as this should have nothing to do with default values in JDBC connectors.
Simply, we should INSERT only to columns which user mentioned in their INSERT query.
What happens beyond that (eg defaulting) should happen in the connector or in RDBMS.

Rename to eg

```
Pass information about columns being inserted to connector
```